### PR TITLE
Add basic GitHub templates

### PR DESCRIPTION
**Pull request by LucasCarioca**
Hey I was the guy that responded about possibly contributing back. 

I figured I would go ahead and start off by helping with some basic GitHub templates to make collecting issues, requests and PRs a little cleaner and more organized. 

I submitted my thoughts on a few templates tailored a bit for a  swift project. Let me know your thoughts.

You can see what these will look like in my fork https://github.com/LucasCarioca/ToastUI/issues/new/choose

Screen Shot 2020-08-10 at 9 49 24 PM

### CODE REVIEW

**Comment by quanshousio** [comment]
Thank you so much for your first contribution. I truly appreciate it. I've read all the templates and have some thoughts:

- `but_report_template.md` has a typo. It should just be `bug_report.md` to be consistent with the name that GitHub sets by default when you create the template.
- Add a full stop to all inline comments that consist of more than one sentence.

**Comment by LucasCarioca** [comment]
Alright sounds good ill make the file name change. 

Not sure what you mean by this though: 
> * Add a full stop to all inline comments that consist of more than one sentence.

File: .github/ISSUE_TEMPLATE/but_report_template.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+<!--- Provide a set of steps to -->
+<!--- reproduce this bug. Include code, configuration or screenshots to reproduce, if relevant -->
+
+
+## Context
+
+<!--- How has this issue affected you? What are you trying to accomplish? -->
+<!--- Providing context (e.g. links to configuration settings, -->
+<!--- stack trace or log data) helps us come up with a solution that is most useful in the real world -->
```
Comment (quanshousio) [inline .github/ISSUE_TEMPLATE/but_report_template.md]: Add period after `in the real world`.

Comment (quanshousio) [inline .github/ISSUE_TEMPLATE/but_report_template.md]: You should add period after `addition or change`, `if relevant` and `in the real world` comments as those are block comments. It's just my preference.

```suggestion
## Possible Solution

<!--- Not obligatory, but suggest a fix/reason for the bug, -->
<!--- or ideas as to the implementation of the addition or change. -->

## Steps to Reproduce (for bugs)

<!--- Provide a set of steps to -->
<!--- reproduce this bug. Include code, configuration or screenshots to reproduce, if relevant. -->

## Context

<!--- How has this issue affected you? What are you trying to accomplish? -->
<!--- Providing context (e.g. links to configuration settings, -->
<!--- stack trace or log data) helps us come up with a solution that is most useful in the real world. -->
```

**Comment by quanshousio** [comment]
> * Add a full stop to all inline comments that consist of more than one sentence.

I meant block comments, sorry for being wordy here. I just made suggested changes in a211e19.

Moreover, I'm seeing `bug_report.md` still has typo in its filename in the last commit. Could you fix that?

File: .github/ISSUE_TEMPLATE/but_report_template.md (hunk 1 above)
Comment (LucasCarioca) [reply .github/ISSUE_TEMPLATE/but_report_template.md]: Fair enough. fixed in latest commits

**Comment by LucasCarioca** [comment]
Oops totally overlooked that. Fixed as well. 

> Moreover, I'm seeing `bug_report.md` still has typo in its filename in the last commit. Could you fix that?

**Comment by LucasCarioca** [comment]
Anymore changes on this @quanshousio?

**Comment by quanshousio** [comment]
@LucasCarioca Apologies for my tardy response. Looks all good to me! Once again, thank you so much for your suggestion and contribution.

**Review by quanshousio — COMMENTED** [review]
@LucasCarioca Apologies for my tardy response. Looks all good to me! Once again, thank you so much for your suggestion and contribution.

**Pull request merged**